news/_posts/2022-04-07-contributor-interview-daniel-hlavacek.md

```diff
@@ -0,0 +1,84 @@
+
+I also developed our complete custom ERP system that allows us to manage all our stores and orders in one place: packing from handheld devices, shipping and label printing, package delivery, tasks, supplier data sources, etc.
+
+**When did you get involved in contributing to the PrestaShop project?** 
+
+We launched our first PrestaShop store in 2014 on the 1.6 version. Back then, I only reported some bugs on Forge[^1] .
+
+In 2018, PrestaShop [moved issues to Github](https://build.prestashop.com/news/moving-issues-to-github/), and around that time, we migrated to the 1.7 version. During the migration to 1.7, I encountered many bugs, so I reported them on Github. When I realized that the PrestaShop team couldn’t fix them all, I motivated myself and decided to contribute. So, I stopped complaining and took action.
+
+[^1]: <em> Former web-based collaborative software platform based on Jira. </em>
+
+**As a freelancer, why do you contribute to the open-source project?**
+
+Honestly? So I have less work when upgrading my stores. The more things I fix, the less work will everybody have when upgrading. I also like the feeling of doing something good: new merchants who create a PrestaShop store will get much better software than the one I started with.
+
+Also, if something is implemented in the core, all modules will count on it and automatically include it in their data. For example, the MPN field: I added it to my PrestaShop build, but all other modules needed to be modified: XML export to Google, marketing modules, etc.
+
+I learned so much by contributing to the open-source project. I was a beginner in PHP back then. I could write some basic things and mostly submitted one-line pull requests. After 5 years, I think I can do almost anything and make complex pull requests on stuff I was afraid to touch, such as the faceted search and others.
+
+**You also take part in the translation of PrestaShop into Czech. What motivates you?**
+
+Translations needed to be improved. There weren’t many active Czech translators and I couldn’t get in contact with any of them. I studied translation and interpretation in high school, so I thought I’d give it a hand! It’s nice to see that PrestaShop is now fully available in Czech. 
+
+**Do you have any advice for first-time contributors?**
+
+First of all, I think the process could be improved. It would be nice to have more people approving and checking pull requests. Right now, if I submit a pull request, it can take several days before I get an answer. Then there are a lot of back and forths and different people taking turns. In the end, it can take a long time for a pull request to be merged and that's a bit discouraging. I think it’s a governance issue. 
```

Comment:
Looks like this is not direct advice to first-time contributors but more like a pain point he encounter while contributing, advice would have been written like: "Merging your pull request will take a lot of time, don't lose patience"

Reply:
Ok, I will read it entirely 😆

Reply:
😄